index.html

```diff
@@ -119,264 +119,156 @@
       </p>
-    </section>
-
-    <!-- Use Cases -->
-    <section id="profile-use-cases">
-      <h2>Use Cases and Requirements</h2>
       <p>
-        A set of over 30 use cases for the Web of Things were contributed by stakeholders
-        from multiple industries for various application domains.
-        These have been published in the WoT Use Cases and Requirements document [[wot-usecases]].
+        The Thing Description specification is designed to be protocol-agnostic 
+        and flexible enough to describe a wide range of existing ("brownfield") 
+        IoT devices, rather than specifying a fixed protocol or application 
+        programming interface (API) which all devices must implement. The downside
+        of this open ended flexibility is that it makes ad-hoc interoperability on
+        the Web of Things very difficult, because it's nearly impossible to create
+        a single WoT <a href="#dfn-consumer">Consumer</a> [[wot-architecture]] 
+        which is guaranteed to be able to communicate with any
+        <a href="#dfn-thing">Web Thing</a> [[wot-architecture]] out of the box.
       </p>
       <p>
-        Based on these use cases a set of requirements have been derived which drive the development
-        of the W3C Web of Things specification family.
-        Several of these domains require easy integration of devices from multiple vendors,
-        in other words, out-of-the-box interoperability.
-        However, the descriptive approach taken by the WoT specifications generally leads
-        to a large variety of different protocols and data formats,
-        which can work against out-of-the box interoperability.
+        This specification is designed to complement the Thing Description 
+        [[wot-thing-description]] specification, by defining a 
+        <a href="#profiling-mechanism">Profiling Mechanism</a> and a set of 
+        <a href="#dfn-profile">Profiles</a> which enable 
+        <a href="#out-of-the-box-interoperability">out-of-the-box
+        interoperability</a> between <a href="#dfn-thing">Web Things</a> and their
+        <a href="#dfn-consumer">Consumers</a>.
       </p>
       <p>
-        For example, a WoT Thing Description (TD) can in theory include a description based on a
-        networking protocol unknown to a device that wishes to connect to it.
-        To ensure interoperability without additional customization (e.g. by writing software or
-        performing complex setup or configuration steps),
-        the range of such choices needs to be limited to a finite set so that a consumer
-        of a Thing Description can be sure it will be able to interact with any possible Thing.
-        A finite set of customization choices is also important for implementing devices with a fixed code base.
-        Defining such constraints leads to the profile mechanism and the HTTP Basic Profile.
+        Profiles are designed specifically for new ("greenfield") implementations 
```

Comment:
```suggestion
```

Reply:
I disagree with removing this statement for the reasons explained above.